<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>radius</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>300</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Area</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="radiusLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>count</string>
   </property>
  </widget>
  <widget class="QLabel" name="areaLabel1_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
